README.md

```diff
@@ -1,2 +1,27 @@
-# .github
-Readme information for TheCookieMC
+## Welcome to TheCookieMC!
+
+![CookieMC banner](https://raw.githubusercontent.com/TheCookieMC/BugReports/main/cookiemc.png?token=GHSAT0AAAAAABZTNNIHRBVS66ROEBB6OHVSY3O5CUA)
+
+Hello and Welcome to the offical github page for TheCookieMC! Here, you will find all of our API's information and documentation!
+
+### What is TheCookieMC?
+
```

Comment:
Shouldn't be a blank line here, not needed.

Reply:
Ok